docker/etna-base-dev/entrypoints/development.sh

```diff
@@ -26,6 +26,16 @@ if [ -n "$UPDATE_STATE" ]; then
 eval "${app_name_capitalized}_ENV=test ./bin/${APP_NAME} migrate"
 fi
 
+if [ -n "$FULL_BUILD" ]; then
```

Comment:
I'm not sure this is the right fix, development.sh invokes `build.sh` which should be running this, which makes me think there is a reason why I don't do this.

I think the correct solution may be to remove the RELEASE_TEST instead from the run-image-test weirdly.

Reply:
Hm...k, let me try that...